src/Listener/RelatedModelsListener.php

```diff
@@ -87,7 +87,7 @@ public function publishRelatedModels($action = null, $entity = null)
         foreach ($models as $name => $association) {
             list(, $associationName) = pluginSplit($association->name());
             $viewVar = Inflector::variable($associationName);
-            if (array_key_exists($viewVar, $controller->viewVars)) {
+            if (array_key_exists($viewVar, (array)$controller->viewVars)) {
```

Comment:
its unnecessary and looks bloated compared to the original code, there is more to read and understand, even if the code was the same, i would still prefer the human-readable optimized version over the explicit cast :)